packages/waku/src/lib/builder/build.ts

```diff
@@ -112,13 +112,14 @@ const analyzeEntries = async (
       }
     }
   }
+  // const analyzeInlineConfig =
   await buildVite({
     plugins: [rscAnalyzePlugin(clientFileSet, serverFileSet)],
     ssr: {
       target: 'webworker',
       resolve: {
-        conditions: ['react-server', 'workerd'],
-        externalConditions: ['react-server', 'workerd'],
+        // conditions: ['react-server', 'workerd'],
```

Comment:
@dai-shi What was the reason behind this before? 

Now I removed it and it worked and fixed the issue as I explained the PR description.

Reply:
is this test still ok:
https://github.com/dai-shi/waku/blob/44007502ccf0a6a9c52eb14f148effec7232ca64/e2e/ssr-target-bundle.spec.ts

Reply:
add this `vite.config.ts` and the build problem is gone but then it will break on start :-(
```ts
import { defineConfig } from 'vite';

export default defineConfig(() => ({
  ...({
    ssr: {
      external:['swr']
    },
  }),
  
}));
```

Reply:
> @dai-shi What was the reason behind this before?
> 
> Now I removed it and it worked and fixed the issue as I explained the PR description.

Hm, I think we need "react-server" condition. It's the purpose of `analyzeEntries`. Not sure if it's the ideal way, but we walk the module graph and find "use client" and "use server". Without the "react-server" condition we would be using a **different** module graph from what we would be using for bundling.

Reply:
☝️ I think that's my misunderstanding. They can't be the same module graph, because `"use client"` might only appear in non-"react-server" condition.

Let me add a commit to run analyze twice.